tradeexecutor/visual/single_pair.py

```diff
@@ -147,6 +147,9 @@ def visualise_technical_indicators(
     for plot_id, plot in visualisation.plots.items():
         df = export_plot_as_dataframe(plot, start_at, end_at)
         if len(df) > 0:
+            
+            line_shape = "hv" if plot.name == "Stop Loss" else "linear"
```

Comment:
Can you make this an explicit function parameter, as setting value based on a name is not a robust solution?